Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -2332,7 +2332,6 @@ def visit_ClassDefNode(self, node):
 
 
 class AnalyseExpressionsTransform(CythonTransform):
-
```

Comment:
I promise this line removal is undone fixed on the version on my hard-disk. It just doesn't seem worth rerunning the tests for it until there are other changes.